.github/copilot-instructions.md

```diff
@@ -0,0 +1,103 @@
+  - For Blazor's Razor components mentioned in article text, use backticks around the name of the component (example: `Counter` component)
+  - For any new or updated .md file, ensure the standard frontmatter (metadata) is included as specified in [Metadata for Microsoft Learn documentation.](https://learn.microsoft.com/en-us/contribute/content/metadata)
+  - For any new or updated .md file added to the repository, ensure the following frontmatter (metadata) is included:
+    - Metadata `ai-usage: ai-assisted` if any AI assistance was used
+    - The correct order of metadata lines is to place the title (`title`) first and the rest of the metadata lines in alphabetical order. Example: `title`, `author`, `description`, `monikerRange`, `ms.author`, `ms.custom`, `ms.date`, `uid`, `zone_pivot_groups`
+    - Metadata `ms.date: <today's date>` with a format of MM-DD-YYYY.  If the file already has a `ms.date` metadata, update it to today's date if more than 50 characters are changed in the file.
+    
+### Version Targeting Common Range Patterns
+- Fixed Range: `>= aspnetcore-7.0 <= aspnetcore-9.0`
+- Open Upper Bound: `>= aspnetcore-7.0`
+- Open Lower Bound: `<= aspnetcore-9.0`
+- Specific Version: `== aspnetcore-9.0`
+
+### Code Snippets
+- For code snippets longer than 6 lines:
+  1. Create a subfolder named after the document the snippet supports.
+  1. Create a `snippets` folder inside that subfolder.
+  1. For the code file:
+     - If the snippet is not version-specific, place the code in a file with the appropriate extension (for example, `.cs` for C#) in the `snippets` folder.
+     - If the snippet is version-specific:
+        1. Create a subfolder inside the `snippets` folder named for the version (for example, `9.0` for .NET 9 or ASP.NET Core 9).
+        1. Place the code in a file with the correct extension inside the version subfolder.
+        1. Add a project file (`.csproj`) to the version subfolder targeting the matching .NET version, if necessary to run or build the snippet.
+- Reference snippets using triple-colon syntax:
+  - **Use file-relative paths** for snippets specific to a single doc file:
+    ```
+    :::code language="csharp" source="../snippets/my-doc/Program.cs":::
+    ```
+  - **Use repository root-relative paths** for shared snippets:
+    ```
+    :::code language="csharp" source="~/tutorials/min-web-api/samples/9.x/todoGroup/TodoDb.cs":::
+    ```
+- For longer snippets, highlight specific lines:
+  ```
+  :::code language="csharp" source="~/path/to/file.cs" range="5-10" highlight="2-3":::
+  ```
+- Use the latest, non-preview C# coding patterns in non-preview code examples
+- Use the latest preview C# coding patterns in preview code examples
+- Use the following code language and indentation standards for markdown code blocks or the `language` attribute of code snippets:
+
+  Content of the snippet | Code language | Indentation in spaces
```

Comment:
Language code sounds better to me than code language.

Reply:
Updated thanks.